lib/dogapi/facade.rb

```diff
@@ -529,6 +530,49 @@ def revoke_screenboard(board_id)
       @screenboard_svc.revoke_screenboard(board_id)
     end
 
+    #
+    # SYNTHETICS
+    #
+    def create_synthetics_test(type, config, options = {})
+      @synthetics_svc.create_synthetics_test(type, config, options)
+    end
+
+    def edit_synthetics_test(test_id, type, config, options = {})
```

Comment:
We should try to keep the library consistent for its users:
```console
$ git grep 'def update' | wc -l
28
$ git grep 'def edit' | wc -l
0
```

`s/def edit_/def update_/`